src/webots/engine/WbSimulationWorld.cpp

```diff
@@ -165,14 +172,15 @@ WbSimulationWorld::~WbSimulationWorld() {
 
 void WbSimulationWorld::step() {
   WbPerformanceLog *log = WbPerformanceLog::instance();
-  if (log)
+  if (log) {
     log->stepChanged();
+    log->lapTime(WbPerformanceLog::SPEED_FACTOR);
```

Comment:
Why not merging `lapTime` and `stepChanged` functions?

Reply:
You're right, the speed factor is tied to the step change and there's no other uses for the `lapTime` function so there's no need to make a function for it.